packages/react/src/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -253,13 +310,20 @@ const ToggleSwitch = React.forwardRef<HTMLButtonElement, React.PropsWithChildren
         sx={sxProp}
         {...rest}
       >
+        {isLoadingLabelVisible ? (
+          <VisuallyHidden>
+            <AriaStatus announceOnShow id={loadingLabelId}>
+              {loadingLabel}
+            </AriaStatus>
+          </VisuallyHidden>
+        ) : null}
         {loading ? <Spinner size="small" /> : null}
```

Comment:
One quick question for this, is the label that we're seeing as "Loading" that's already in the component coming from Spinner? If so, should we try to coordinate between the two so that one component "owns" the loading label?

Reply:
Good point! We'd only want to rely on the one a consumer can provide directly to `ToggleSwitch`. I adjusted `Spinner` so that it will no longer say "loading" anymore.